modules/loading-animation/css/lib/loading-dots.scss

```diff
@@ -18,35 +18,36 @@
   }
 }
 
-.wdc-loading-spinner::after,
-.wdc-loading-spinner span::before,
-.wdc-loading-spinner span::after {
+.wdc-loading-dots::after,
+.wdc-loading-dots span::before,
+.wdc-loading-dots span::after {
   background-color: $wdc-color-soap-400;
   width: $wdc-spacing-s;
   height: $wdc-spacing-s;
   display: block;
   position: absolute;
   font-size: 0px;
   content: ' ';
-  top: 50%;
-  left: 50%;
+  // top: 50%;
+  // left: 50%;
```

Comment:
do we still want these commented out?

Reply:
Whoops, no we don't :) Careless error, I'll remove the commented-out styles.

Reply:
all good! Just checking :)

Reply:
Thanks for the catch!